src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/RemoveValidator.java

```diff
@@ -3,25 +3,73 @@
 import org.springframework.stereotype.Component;
 import uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto;
 import uk.gov.companieshouse.overseasentitiesapi.model.dto.RemoveDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.UpdateDto;
 import uk.gov.companieshouse.overseasentitiesapi.utils.ApiLogger;
 import uk.gov.companieshouse.overseasentitiesapi.validation.utils.ValidationMessages;
 import uk.gov.companieshouse.service.rest.err.Errors;
 
+import java.time.LocalDate;
+
 import static uk.gov.companieshouse.overseasentitiesapi.validation.utils.UtilsValidators.setErrorMsgToLocation;
 import static uk.gov.companieshouse.overseasentitiesapi.validation.utils.ValidationUtils.getQualifiedFieldName;
 
-
 @Component
 public class RemoveValidator {
 
     public Errors validate(RemoveDto removeDto, Errors errors, String loggingContext) {
-        if (Boolean.FALSE.equals(removeDto.getIsNotProprietorOfLand())) {
-            String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.REMOVE_FIELD, RemoveDto.IS_NOT_PROPRIETOR_OF_LAND_FIELD);
-            String errorMessage = ValidationMessages.NOT_VALID_ERROR_MESSAGE.replace("%s", qualifiedFieldName);
-            setErrorMsgToLocation(errors, qualifiedFieldName, errorMessage);
+
+        // TODO Check that the filing date is 'null' if an Update block is present. See UAR-1461.
+
+        if (removeDto.getIsNotProprietorOfLand() != null) {
+            validateRemoveStatement(removeDto.getIsNotProprietorOfLand(), errors, loggingContext);
+        }
+
+        return errors;
+    }
+
+    public Errors validateFull(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
```

Comment:
Note that I followed the same pattern as already present in the `UpdateValidator` for this implementation.